packages/hash/frontend/src/components/layout/PageSidebar/AccountPageList/PageTreeItem.tsx

```diff
@@ -105,16 +110,13 @@ const CustomContent = React.forwardRef((props: TreeItemContentProps, ref) => {
       >
         <Typography
           variant="smallTextLabels"
+          className="page-title"
```

Comment:
Is this the recommended way of approaching styles in MUI? Just wonder if there can be anything more robust than `"& .page-title"`